### PR TITLE
Save and Load functionality

### PR DESCRIPTION
**Pull request by hanneseilers**
Added functionality to save and load lockly code to file and back to UI.
Works with blockly only (python can be saved with text copy and paste :-))

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-.DS_Store
```
Comment (silbo) [inline .gitignore:1]: @hanneseilers could we leave this still in for Mac users : )

File: assets/js/main.js [hunk 2]
```diff
@@ -17,6 +17,74 @@ let peerjsInitalized = false;
 // To remember last pressed button
 let lastPressedStart = false;
 
+// Save code to file
+window.saveCodeToFile = function saveCodeToFile(){
```
Comment (silbo) [inline assets/js/main.js]: @hanneseilers could you add spaces before the "{", so the code style is consistent. Check all occurrences also below.

**Comment by hanneseilers** [comment]
Requested changes applied.

**Comment by hanneseilers** [comment]
Last version of .gitinroe only ignore idea project folder. everythin was left as it is.
incremented intercace version number to 1.4

**Pull request merged**